LSPS5/README.md

```diff
@@ -0,0 +1,623 @@
+  `lsps5.webhook_registered` notification.
+* MUST remember all webhooks:
+  * If there are currently no channels with this client, for at
+    least 7 days.
+    * If a new channel with the client is opened, then as below.
+  * If there is at least one open channel with this client,
+    indefinitely remember them, until for at least 7 days after
+    the last channel with the client is closed.
+
+On success, the LSP also contacts the webhook and sends the
+`lsps5.webhook_registered` notification without parameters.
+Only the newly-registered webhook is contacted for this
+notification (unlike other events, which contact all
+registered webhooks).
+The LSP MAY send this notification before or after responding to
+the `lsps5.set_webhook` call.
+The LSP MUST send this notification to this webhook before sending
+any other notifications to this webhook.
+This `lsps5.webhook_registered` notification is sent, whether or
+not this is a new registration, even if the client re-registered
+the same `app_name` and `webhook`.
+
+On success, `lsps5.set_webhook` returns an object like the
+following:
+
+```JSON
+{
+  "num_webhooks": 2,
+  "max_webhooks": 4
+}
+```
+
+`num_webhooks` is the number of webhooks already registered,
+including this one if it added a new webhook.
+`max_webhooks` is the maximum number of webhooks the LSP allows
+per client.
+Both are unsigned JSON integers.
+
+The `lsps5.set_webhook` call has the following errors defined
+(error `code` number in parenthesis):
+
+* `too_long` (0) - Either `app_name` or `webhook` or both are
+  too long.
+* `unsupported_protocol` (1) - The `webhook` is not a protocol
+  that the LSP supports.
+  * Future revisions of this LSPS MAY allow other protocols than
+    HTTPS.
+    However, an LSP that does not support that protocol is
+    allowed to return this error, and the client MUST either
+    set up with a protocol that the LSP supports, or not have
+    webhook notifications.
+    An LSP MUST support HTTPS at the minimum.
+* `too_many_webhooks` (2) - The client already has the maximum
+  number of webhooks that the LSP allows.
+  The `data` object of the `error` response contains the field
+  `max_webhooks`, a JSON unsigned integer indicating the
+  maximum number of webhooks supported by the LSP.
+
+#### `lsps5.list_webhooks`
+
+The client can use `lsps5.list_webhooks` to learn all `app_name`s
+that have webhooks registered for the client.
+It takes no parameters `{}`.
+
+`lsps5.list_webhooks` returns an object like the following:
+
+```JSON
+{
+  "app_names": ["My LSPS-Compliant Lightning Wallet", "Another Wallet With The Same Signing Device"]
+}
+```
+
+There are no errors defined for `lsps5.list_webhooks`.
+
+#### `lsps5.remove_webhook`
+
+The client can outright remove an existing registered webhook
+via the `lsps5.remove_webhook` call, which takes the parameters:
+
+```JSON
+{
+  "app_name": "Another Wallet With The Same Signig Device"
+}
+```
```

Comment:
If does not exist, we are happy because it already does not exist, which is the goal of removal?

Hmmm OK seems fine to have an error for this case.

Reply:
Done.